force-app/main/educationCloudSettings/lwc/edcSettingsProductCard/edcSettingsProductCard.html

```diff
@@ -17,38 +17,41 @@ <h3 slot="title" id="cardTitle">
                     </div>
```

Comment:
Hi @eclemmer , I just noticed that prettier is complaining this file is not following the standards. You can install prettier on vscode and it automatically runs prettier when you save the file. Then when you commit and push the changes, this check should pass.

Reply:
@salvadorsfdc This has been updated.

Reply:
awesome! thanks